calculatorFrontend/docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3.8"
```

Comment:
you should be able to move this dockercompose so it starts both docker containers